athf/commands/attack.py

```
@@ -0,0 +1,249 @@
+
+    \b
+    Examples:
+      athf attack update
+      athf attack update --force
+    """
+    try:
+        from mitreattack.stix20 import MitreAttackData
+    except ImportError:
+        console.print("[red]Error: mitreattack-python is not installed.[/red]")
+        console.print("[dim]Install it with: pip install 'athf[attack]'[/dim]")
+        raise click.Abort()
```

Ruff (0.15.6)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)